ext/opcache/jit/zend_jit_ir.c

```diff
@@ -5591,8 +5602,8 @@ static int zend_jit_math_helper(zend_jit_ctx   *jit,
 			if (opline->opcode == ZEND_ASSIGN_DIM_OP && (opline->op2_type & (IS_VAR|IS_TMP_VAR))) {
 				ir_GUARD_NOT(ir_LOAD_A(jit_EG_exception(jit)),
 					jit_STUB_ADDR(jit, jit_stub_exception_handler_free_op2));
-			} else if (Z_MODE(res_addr) == IS_MEM_ZVAL && Z_REG(res_addr) == ZREG_RX) {
-				zend_jit_check_exception_undef_result(jit, opline);
+			// } else if (Z_MODE(res_addr) == IS_MEM_ZVAL && Z_REG(res_addr) == ZREG_RX) {
+			// 	zend_jit_check_exception_undef_result(jit, opline);
```

Comment:
I'm not sure whether this was necessary. `zend_jit_check_exception_undef_result()` will set opline.result to UNDEF (for IS_VAR|IS_TMP_VAR results) before handling exception, I didn't find cases where this was necessary.

I need to check further.